pegasus/sites.v3/code.org/public/css/container-responsive-full-width.css

```diff
@@ -0,0 +1,26 @@
+/* Allows for full-width banners and sections on Pegasus pages */
+
+#pageheader-wrapper #pageheader {
+  margin-bottom: 0;
+}
+
+.container_responsive_full_width {
+  width: 100%;
+  padding: 0;
+}
```

Comment:
it looks like we do a few other styles applied to container_responsive. do we want to replicate those behaviors here? For example, shrinking the size of the headers as the page gets narrower: https://github.com/code-dot-org/code-dot-org/blob/b90a9daa5c1dd102782e8a811989f8529e620af7/pegasus/sites.v3/code.org/public/css/brstrap.css#L12-L15

you can see this in action in a different PR I just happened to be working on right now: https://github.com/code-dot-org/code-dot-org/pull/50034

same question for all other mentions of container_responsive in brstrap.css.

Reply:
The plan is to streamline all the styles when we go site-wide w/ the rebrand where stuff like this can be accounted for. We're trying to isolate the rebranded pages from the non-rebranded pages for now — it's all a work in progress, but will hopefully come together at some point soon!